### ISSUE TITLE
[Bug]: Allow wallet to be unlocked if the selected network is inaccessible

### ISSUE THREAD

**Issue by plasmacorral** [MetaMask/metamask-extension#20384]
### Describe the bug

When the selected network is unavailable prior to v10.33, a user is unable to unlock and there is no ability to change networks from the lock screen.  In 10.33 and beyond, the network selection becomes available on the lock screen and allows a user to switch to another network that is reachable and proceed to unlock.

As an incremental step towards better offline support, we should enable the lock screen (specifically the ability to unlock) to be unaffected by network status.  This would facilitate the removal of network selection from the lock screen entirely.

### Steps to reproduce

1. Select or create a network and connect to it
2. Lock wallet
3. Block selected network at your router's firewall
4. Attempt to unlock without changing networks

### Version

10.34

### Build type

None

### Browser

Chrome, Firefox

### Operating system

MacOS

### Additional context

Related to https://github.com/MetaMask/metamask-extension/issues/5210

**Comment by plasmacorral** [comment]
Still a thing

**Comment by plasmacorral** [comment]
Issue remains in 11.4.0.  User cannot unlock MM if the selected network is unavailable, however, a user can access the networks drop down to select a network that is available and then proceed to unlock.
